geoips/compare_outputs.py

```diff
@@ -244,7 +244,11 @@ def images_match(output_product, compare_product, fuzz="5%"):
     ]
 
     LOG.info("**Running %s", " ".join(call_list))
-    fullimg_retval = subprocess.call(call_list)
+    fullimg_ret = subprocess.run(
```

Comment:
Updated to use `subprocess.run()` rather than `subprocess.call()`. From the documentation for `subprocess.call()`:

> Do not use stdout=PIPE or stderr=PIPE with this function. The child process will block if it generates enough output to a pipe to fill up the OS pipe buffer as the pipes are not being read from.

I also piped stderr to stdout and added this output to `LOG.debug()`.